zebra-consensus/src/block/subsidy/founders_reward.rs

```diff
@@ -26,11 +31,69 @@ pub fn founders_reward(height: Height, network: Network) -> Result<Amount<NonNeg
     }
 }
 
+/// Get the founders reward t-address for the specified block height as described in [protocol specification §7.8][7.8]
+///
+/// [7.8]: https://zips.z.cash/protocol/canopy.pdf#foundersreward
+pub fn founders_reward_address(height: Height, network: Network) -> Result<String, Error> {
```

Comment:
According to the spec(https://github.com/ZcashFoundation/zebra/pull/1129) this should be changed to return a `PayToScriptHash`.

Reply:
Let's also add a test that makes sure each address parses as a `PayToScriptHash`.

We should be able to implement the test using `founders_reward_address` and the address change interval.